static/app/components/replays/canvasReplayerPlugin.tsx

```diff
@@ -314,11 +338,7 @@ export function CanvasReplayerPlugin(events: eventWithTime[]): ReplayPlugin {
         // Only do this when isSync is true, meaning there was a seek, since we
         // don't know where next index is
         nextPreloadIndex = -1;
-
-        if (isCanvas) {
-          processEventSync(e, {replayer});
-        }
-
```

Comment:
Moved the isCanvas check to inside `processEventSync` so that we can trigger `debouncedProcessQueuedEvents()` as we need to know when `handleQueue` is empty as well.